ports/libmypaint/vcpkg.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "libmypaint",
+  "version": "1.6.1",
+  "description": "Brush library used by MyPaint",
+  "homepage": "mypaint.org",
+  "license": null,
```

```suggestion
  "license": "ISC",
```

This COPYING looks like https://spdx.org/licenses/ISC.html ?

---

Also https://github.com/mypaint/mypaint/blob/master/Licenses.md says "The libmypaint component (brushlib/) is licensed under the ISC license."

---

I am not expert in licenses. what the different between ISC and CC0-1.0?
On the repo of mypaint-brushes:
https://github.com/mypaint/mypaint-brushes/blob/master/COPYING

I will change for ISC.

---

I changed to ISC.